netket/vqs/mc_expect.py

```diff
@@ -144,3 +161,34 @@ def log_pdf(w, σ):
     )
 
     return Ō_stats
+
+
+def _expect_continuous(
+    machine_pow: int,
+    model_apply_fun: Callable,
+    kernel: Callable,
+    parameters: PyTree,
+    masses: PyTree,
```

Comment:
OK, I agree with the latest comment from @PhilipVinc (namely, let's try to unify these kernels after this PR) however I am still confused as per why we need to pass masses here explicitly, I think this can be already easily avoided?

Reply:
@gcarleo masses must be passed in some way through the 'jax.jit' barrier to make jax happy.
We can pass the masses implicitly, by passing the whole operator as a pytree, or explicitly, as done here.

To do it implicitly we'd need to make our operators data classes. It's in my plans but I did not have the time to do it yet.

However, @gpescia , this field here should not be called 'masses' but 'additional data' because it can be anything, depending on the operator type. For example, in the case of the sumOperator it is a pytree, for Potential is none, and only for kinetic it is the masses